src/media/js/main.js

```diff
@@ -52,7 +45,7 @@ function() {
     z.page.on('navigate divert', function(e, url) {
         if (url == urls.reverse('login') ||
             url == urls.reverse('unauthorized') ||
-            url == urls.reverse('fxa_authorize')) {
+            url.indexOf(urls.reverse('core/fxa_authorize')) !== -1) {
```

Comment:
let's leave it to pull in later if it makes it up to mcm. i'm not too motivated to improve it from how it is now